tools/docker_dev/.gitpod.Dockerfile

```diff
@@ -0,0 +1,69 @@
+# Using the Scipy-dev Docker image as a base
+# This way, we ensure we have all the needed compilers and dependencies
+# while reducing the build time
+ARG BASE_CONTAINER=scipy/scipy-dev:20210217-master-395b89d6
```

Comment:
I pinned this to the last created SciPy docker image - will add a separate PR to add the update through the gh action workflow